proseco/catalog.py

```diff
@@ -316,9 +316,9 @@ def merge_cats(fids=None, guides=None, acqs=None):
     def filt(tbl, id_set):
         return [row['id'] for row in tbl if row['id'] in id_set]
 
-    bot_ids = filt(guides, set(guides['id']) & set(acqs['id']))
-    gui_ids = filt(guides, set(guides['id']) - set(acqs['id']))
-    acq_ids = filt(acqs, set(acqs['id']) - set(guides['id']))
+    bot_ids = sorted(filt(guides, set(guides['id']) & set(acqs['id'])))
```

Comment:
Put the `sorted` into the `filt` function (with additional comment about sorting by ID within each group to match what Matlab does).

Reply:
Since now it is just sorting the bots, I don't think it makes sense to push into `filt`.